packages/bpk-component-accordion/src/withSingleItemAccordionState.js

```diff
@@ -29,21 +38,31 @@ const getInitiallyExpanded = children => {
   return (result || {}).key || null;
 };
 
-const withSingleItemAccordionState = ComposedComponent => {
-  class WithSingleItemAccordionState extends Component {
-    constructor(props) {
+type Props = {
+  children: Node,
+};
+
+type State = {
+  expanded: ?string | ?number,
```

Comment:
This gave mixed errors for both string and number - so added it to be either or as per the React error being thrown here.